tests/amx/harness_amx.py

```diff
@@ -267,4 +269,4 @@ def stop_timer(self, name, msg):
 
     def add_body(self, lines):
         assert type(lines) is list
-        self.body += lines
+        self.body += lines
```

Comment:
Why did you remove the newline at the end of the file?

Reply:
This I'm not sure. @skeqiqevian ?